dfttopif/parsers/base.py

```diff
@@ -46,16 +51,38 @@ class DFTParser(object):
     _converged = None
     ''' Whether this calculation has converged '''
     
-    def __init__(self, directory):
-        '''Initialize a parser.
+    def __init__(self, directory, files=None):
```

Comment:
Does `directory` still need to be required?  Is it ever safe to use internally if `files` is specified?